vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml

```diff
@@ -96,4 +96,49 @@
       line: "  <httpLogin>{{ ldap_user_prefix }}arkcase-admin</httpLogin>"
     - regexp: "<httpPassword>Encrypted 2be98afc81ac2978aac22ab798bb3aac0</httpPassword>"
       line: "  <httpPassword>{{ arkcase_admin_encr_password.stdout }}</httpPassword>"
-      
\ No newline at end of file
+
+- name: run the job to create and populate the mondrian cubes
+  become: yes
+  become_user: pentaho-pdi
+  command: 
+    cmd: ./kitchen.sh -file://{{ root_folder }}/install/pentaho/foia-reports-dw-{{ foia_analytical_reports_version }}/foia-dw1.kjb
+    chdir: "{{ root_folder }}/app/pentaho-pdi/data-integration"
+
```

Comment:
@david-oc-miller, you are using "kitchen" instead of a carte server, are we good if we can not monitor, start, stop the job from the Carte UI?

Reply:
@marjan4jc we can use cron to schedule the kitchen.sh job, with log analysis to monitor the daily cron job.  I don't see a need for a carte server in production ... we shouldn't have manual sysadmin tasks in a prod environment. 

That being said I do need to add such a cron job.

Reply:
@david-oc-miller , why we need to schedule the kitchen job, there is a scheduler configured in the job itself already, it will run the transformations each day at 2am. When kitchen.sh start the job it will stay started and re-run the transformations each day.

Reply:
cron is a dependable scheduler and we don't need either a running kitchen.sh nor a running carte server ... In my opinion Pentaho has not written the most reliable software in the world, so I'd rather use the operating system scheduler to run a plain shell script.

Reply:
@david-oc-miller, will need to remove the scheduler in the job. Because if we start it like it is, the process will not stop until we stop it, and will not run the transformations for the first time until  2am.

Reply:
@marjan4jc , this is not what I have observed; when running kitchen.sh in this way, it does create all the tables, runs the cache clean jobs, and then exits.

Reply:
@david-oc-miller, probably on the ftp you have a version where the scheduler in the job is not set.

Reply:
hmm... I have just taken the master branch as of 1-July-2020.  How can I tell by looking at the kjb file?

Reply:
yes, I just checked the master branch, and it is without scheduler ... will stay that way. If you want to be sure if it is not present, you can check the .kjb file for this line "`<repeat>N</repeat>`", if here we have N, we do not have a scheduler.

Reply:
@marjan4jc, I updated this code to be sure to replace `<repeat>Y</repeat>` with `<repeat>N</repeat>`.  Also I added a cron job to run the job at 2 AM daily.